content/en/docs/reference/using-api/client-libraries.md

```diff
@@ -64,6 +64,7 @@ their authors, not the Kubernetes team.
 | Ruby                 | [github.com/abonas/kubeclient](https://github.com/abonas/kubeclient) |
 | Ruby                 | [github.com/kontena/k8s-client](https://github.com/kontena/k8s-client) |
 | Rust                 | [github.com/ynqa/kubernetes-rust](https://github.com/ynqa/kubernetes-rust) |
+| Rust                 | [github.com/clux/kube-rs](https://github.com/clux/kube-rs) |
```

Comment:
Hi @clux 
I think this should sort before `github.com/ynqa/kubernetes-rust`

No big deal though!
/lgtm

Reply:
Reordered it!